app/v2/controllers/request/BodyReplaceableRequests.scala

```diff
@@ -26,4 +27,5 @@ abstract class BodyReplaceableRequest[+R[_], B](request: Request[B]) extends Wra
 
 case class AuthenticatedRequest[A](eoriNumber: EORINumber, request: Request[A]) extends BodyReplaceableRequest[AuthenticatedRequest, A](request) {
   override def replaceBody(body: A): AuthenticatedRequest[A] = AuthenticatedRequest(eoriNumber, request.withBody(body))
+  val acceptHeader                                           = request.headers.get(HeaderNames.ACCEPT)
```

Comment:
Is this needed now? I suspect this is a bit of a hangover from an earlier iteration -- the only place that this is used as far as I can see is once in the controller -- we might as well just get it via `request.headers.get(HeaderNames.ACCEPT)` there.

If you feel it is needed, then fair enough, just not convinced this is!